cmd/zpool/zpool_main.c

```diff
@@ -2379,6 +2380,10 @@ print_status_config(zpool_handle_t *zhp, status_cbdata_t *cb, const char *name,
 			(void) printf(gettext("all children offline"));
 			break;
 
+		case VDEV_AUX_BAD_LABEL:
+			(void) printf(gettext("invalid label"));
+			break;
```

Comment:
It seems like we should add this same case to `print_import_config()` so you'll get the same result from `zpool import`  when scanning for exported pools which can be imported.